infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -185,8 +182,10 @@ public LogReplicationEntry processAcks() throws InterruptedException, ExecutionE
      * @param message
      */
     public CompletableFuture<LogReplicationEntry> sendWithBuffering(LogReplicationEntry message) {
-        message.getMetadata().setSnapshotSyncSeqNum(maxAckForSnapshotSync++);
+        //todo need to be removed.
+        message.getMetadata().setSnapshotSyncSeqNum(maxAckedMsgSeqNum++);
         pendingMessages.append(message);
+        log.info("Send messsage {}", message.getMetadata());
```

Comment:
I think we can remove this or downgrade to trace as we already log right before sending.